r/tests/testthat/setup.R

```diff
@@ -0,0 +1,3 @@
+library(httptest)
+library(Seurat)
+library(dplyr)
```

Comment:
moved these here to follow [best practices](https://testthat.r-lib.org/reference/test_dir.html#special-files)